bin/razor-admin

```diff
@@ -78,7 +86,7 @@ end
 
 # It is very important that razor/initialize isn't loaded until after
 # RACK_ENV has been set
-require_relative '../lib/razor/initialize'
+require 'razor/initialize'
```

Comment:
Oh no, I remember now. [Jruby implemented require_relative slightly differently than MRI](https://github.com/jruby/jruby/issues/3092). It's probably fixed now, but it was squishy behavior in the spec that led to my thoughts on not using require relative.